### ISSUE TITLE
`import InteractiveUtils` in `startup.jl` breaks default behavior

### ISSUE THREAD

**Issue by goretkin** [JuliaLang/julia#40192]
The `isdefined` check here returns `true` if there is an `import InteractiveUtils` in `startup.jl`
https://github.com/JuliaLang/julia/blob/188755071fa372ba9c6fb0c11f316f075fbe3ddd/base/client.jl#L357

The behavior of `isdefined` is expected, but the behavior of `load_InteractiveUtils` is not. Is the check even needed?

**Comment by KristofferC** [comment]
So to be clear, the point is that e.g the unqualified `@code_warntype` will stop working in the REPL because the `using` statement will not be run?

**Comment by goretkin** [comment]
> e.g the unqualified `@code_warntype` will stop working in the REPL because the `using` statement will not be run

Yes, exactly. same with e.g. `@edit`, `@which`, etc. Thank you for clarifying.

**Comment by goretkin** [comment]
Is it necessary to check `isdefined(Main, :InteractiveUtils)`? At a glance, it seems like that code could run unconditionally. 

If there does need to be a check, it should be a different check.
